File: Week03/C++/task3.cpp
#include <iostream>
using namespace std;

int main() {

    int n;

    cout << "Lutfen 3 ile 9 arasinda bir sayi girin: ";
    cin >> n;

    while (n < 3 || n > 9) {
        cout << "Gecersiz sayi.Lutfen 3 ile 9 arasinda bir sayi girin: ";
        cin >> n;
    }

    for (int i = 1; i <= n; i++) {

        for (int j = 1; j <= i; j++) {
            cout << j;
        }

        cout << endl;
    }

    for (int i = n - 1; i >= 1; i--) {

        for (int j = 1; j <= i; j++) {
            cout << j;
        }

        cout << endl;
    }
}

File: Week03/Python/task3.py
n = int(input("Lütfen 3 ile 9 arasında bir sayı girin: "))

while n < 3 or n > 9:
    n = int(input("Lütfen 3 ile 9 arasında bir sayı girin: "))

k = 1
direction = 1

while k > 0:
    i = 1
    while i <= k:
        print(i, end="")
        i += 1

    print()

    if k == n:
        direction = -1

    k += direction